bee-node/src/cli.rs

```diff
@@ -45,7 +40,15 @@ struct Cli {
     commit_version: bool,
     // Whether the node should run as an (autopeering) entry node.
     #[structopt(long = "entry_node", help = "Runs as autopeering entry node")]
```

Comment:
Is this the CLI argument name we would actually have to use ? As in `--entry_node` ? If yes, we should probably use a dash instead of an underscore.

Reply:
I agree with Thibault here

Reply:
I absolutely agree as well

Reply:
Fixed